charts/vald/values.yaml

```diff
@@ -995,6 +995,9 @@ gateway:
       # @schema {"name": "gateway.vald.ingress.servicePort", "type": "string"}
       # gateway.vald.ingress.servicePort -- service port to be exposed by ingress
       servicePort: grpc
+      # @schema {"name": "gateway.vald.ingress.usev1beta1", "type": "boolean"}
+      # gateway.vald.ingress.usev1beta1 -- use networking.k8s.io/v1beta1 instead of v1
+      usev1beta1: false
```

Comment:
Currently the default value is `false`. what do you think about it?

Reply:
I think it's okay, and we should add comment if k8s 1.22 released it would be better to remove this option